src/Screens/Logs/Log.tsx

```diff
@@ -314,12 +314,12 @@ const Log: FunctionComponent<LogProps> = ({ navigation, route }) => {
                       <View style={styles.divingGearLabelContainer}>
                         <View style={styles.airTankLabelContainer}>
                           <Text style={styles.airTankLabelText}>
-                            {t('AIR_TANK')} 1
+                            {t('AIR_TANK')}
                           </Text>
                         </View>
                         <View style={styles.airTankMeasurementContainer}>
                           <Text style={styles.capacityText}>
-                            {airLimit}{' '}
+                            {diveLog.review.start_air}{' '}
```

Comment:
Can you include a screenshot of how this page looks now?

Reply:
<img width="413" alt="image" src="https://user-images.githubusercontent.com/63934597/170386293-c9597ff7-00bb-4043-b6f1-b3ebb4c8faf4.png">

Reply:
i decided to change the design, from how it was in the original design

Reply:
Got it, thank you.